src/Trip.js

```diff
@@ -11,14 +11,22 @@ class Trip {
   }
 
   calculateCost(destinations) {
-    let totalSpent;
+    let totalSpent = 0;
     const destination = destinations.find(destination => destination.id === this.destinationID);
     totalSpent += destination.estimatedFlightCostPerPerson * this.travelers;
     totalSpent += destination.estimatedLodgingCostPerDay * this.duration;
-    return totalSpent
+    return this.commafy(totalSpent)
+  }
+
+  commafy(cost) {
```

Comment:
I like the the use of this function! That's a good way to bring readability to these large numbers.